src/components/containers.js

```diff
@@ -135,3 +136,120 @@ export const IndicatorsContainer = (props: IndicatorContainerProps) => {
     </Div>
   );
 };
+
+// ==============================
+// Value Spacer
+// ==============================
+
+export type ValueSpacerProps = CommonProps & {
+  /** Set when the value container should hold multiple values. This is important for styling. */
+  placeholder?: string,
+  options?: OptionsType,
+  children: Node,
+  getOptionLabel: Function,
+  formatOptionLabel: Function,
+  isDisabled: boolean,
+  components: SelectComponents
+};
+export const valueSpacerCSS = () => ({
+  display: 'flex ',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  margin: 0,
+  padding: 0,
+  height: 0,
+  visibility: 'hidden',
+});
+
+export class ValueSpacer extends Component<ValueSpacerProps>{
+  longestOption: OptionType | null;
+  componentWillMount () {
+    this.longestOption = this.getLongestOption(this.props.options);
+  }
+
+  componentWillUnmount () {
+    this.longestOption = null;
+  }
+
+  componentDidUpdate (prevProps: ValueSpacerProps) {
+    if (prevProps.options !== this.props.options) {
+      this.longestOption = this.getLongestOption(this.props.options);
+    }
+  }
+
+  flattenOptions = (options: OptionsType) => {
+    return options.reduce((acc, current) => {
+      if (Array.isArray(current.options)) return [...acc, ...current.options];
+      return [...acc, current];
+    }, []);
+  }
+
+  getLongestOption = (options: OptionsType) => {
+    const { getOptionLabel } = this.props;
+    const sortedOptionsList = this.flattenOptions(options).sort((a, b) => getOptionLabel(a).length - getOptionLabel(b).length);
+    return sortedOptionsList[sortedOptionsList.length - 1];
+  }
+
+  render () {
+    const {
+      children,
+      isDisabled,
+      formatOptionLabel,
+      placeholder,
+      components,
+      ...commonProps
+    } = this.props;
+
+    // $FlowFixMe
+    const { getStyles } = commonProps;
+    const { Placeholder, SingleValue } = components;
+    return (
+      <div>
+        <Div
+          css={getStyles('valueSpacer', this.props)}
+        >
+          {placeholder &&
```

Comment:
here we render the placeholder still, but no longer render every option as a singleValue.